core/src/test/java/org/apache/iceberg/view/ViewCatalogTests.java

```diff
@@ -99,7 +100,7 @@ public void basicCreateView() {
                 .timestampMillis(view.currentVersion().timestampMillis())
                 .versionId(1)
                 .schemaId(0)
-                .putSummary("operation", "create")
+                .summary(view.currentVersion().summary())
```

Comment:
we could also use `EnvironmentContext.get()` here for comparison, but I'm not sure if this might later cause test issues when view support is added for other engines